test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | hot                     |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+
+  @maximum_size
+  Scenario:  try to create a new entity NGSI v2 with maximun size in payload (8972 attributes = 1048576 bytes)
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_maximum_size |
+      | Fiware-ServicePath | /test             |
+      | Content-Type       | application/json  |
+    When create "1" entities with "8972" attributes
+      | parameter        | value     |
+      | entities_type    | room      |
+      | entities_id      | room2     |
+      | attributes_name  | max_size  |
+      | attributes_value | random=10 |
+      | attributes_type  | my_type   |
+      | metadatas_number | 1         |
+      | metadatas_name   | very_hot  |
+      | metadatas_type   | alarm     |
+      | metadatas_value  | random=10 |
+    Then verify that receive several "Request Entity Too Large" http code
+    And verify several error responses
+      | parameter   | value                                              |
+      | error       | Request Entity Too Large                           |
+      | description | payload size: 1048647, max size supported: 1048576 |
+    And verify that entities are not stored in mongo
+
+  @content_type_without
+  Scenario:  try to create entities in NGSI v2 without content-type header
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_content_type |
+      | Fiware-ServicePath | /test             |
+    When create "1" entities with "1" attributes
+      | parameter        | value       |
+      | entities_type    | room        |
+      | entities_id      | room2       |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+    Then verify that receive several "Unsupported Media Type" http code
+    And verify several error responses
+      | parameter   | value                                                                           |
+      | error       | Unsupported Media Type                                                          |
+      | description | Content-Type header not used, default application/octet-stream is not supported |
+
+  @content_type_error
+  Scenario Outline:  try to create entities in NGSI v2 with wrong content-type header
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_content_type |
+      | Fiware-ServicePath | /test             |
+      | Content-Type       | <content_type>    |
```

Comment:
This means literarly a heade containint `<content_type>`. Maybe something more explicit could be better (e.g. "foo/bar").